decisions/Gondor.txt

```diff
@@ -1785,6 +1785,23 @@ political_decisions = {
         effect = {
             prestige = 10
             badboy = 3
+			random_owned = {
+				limit = {
+					owner = {
+						HAR = {
+							exists = yes
+							vassal_of = GON
+						}
+					}
+				}
+				owner = {
+					HAR_286 = { add_core = HRN }
+					HAR = { release = HRN }
+					GON = {	inherit = HRN }
+					HAR_286 = { remove_core = HRN }
```

Comment:
What's up with adding and removing cores in the same decision from the same region without doing something in the middle?

Reply:
Did you not see the two release an annexation lines? I want to make sure that Ninzayan is an HRN core because forming HAR removes individual tribes' cores.

Reply:
You're right. Totally missed that